### DISCUSSION TITLE
Welcome to pagesigner Discussions!

### DISCUSSION THREAD

**Discussion by themighty1** [tlsnotary/PageSigner#47] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by farazshaikh** [comment]
Hello 

how is pagesigner different from the Deco protocol ? Can it be used to certify data received from a rest endpoint ?

Thanks

**Reply by themighty1** [reply]
Hey @farazshaikh , it is different in the underlying implementation. The end goal is the same for DECO and TLSNotary.

**Reply by themighty1** [reply]
Yes, data received from an endpoint can be certified, as long the the third-party who receives the certification trusts that the notary is honest. PageSigner uses a notary in a sandboxed environment which cannot be tampered with.
